WIP Custom Avatar Redesign (#654)
vck3000/ProAvalon: README.md

### Running

1. Start the database and MinIO client with: `docker-compose up`. Ensure Docker is running.
2. Start the server with: `yarn dev`.

### Setting up MinIO

1. Access MinIO via localhost:9001

> 2. Create an access key and paste its details into `.env`

Reviewer: ```suggestion
2. Create an access key and paste the key id into `AWS_ACCESS_KEY_ID` and access key into `AWS_SECRET_ACCESS_KEY` in the  `.env`'s 
```

1. Access MinIO via localhost:9001
2. Create an access key and paste the key id into `AWS_ACCESS_KEY_ID` and access key into `AWS_SECRET_ACCESS_KEY` in the `.env` file.

> 3. Create a Bucket `proavalon` in MinIO. After creation, set its Access Policy to Public.

Reviewer: ```suggestion
3. Create a Bucket named `proavalon` in MinIO and set its Access Policy to Public.
```

Reviewer: Or, split the Set Access Policy step to step 4.
